configuring_project.md

```diff
@@ -72,9 +69,9 @@ build_flags =
 
 ## Configuring defines.h
 
-To use this project with your specific configuration, you need to make changes in the `defines.h` file.
+The `defines.h` file specifies the information about your IMU.
```

Comment:
This isn't strictly true, it's an overal setup configuration, this can also change depending on the MCU.

Reply:
Would it be correct to include it like this then: `information about your IMU and MCU`? Not sure about the specifics, but I see that defines the information about IMU, and for MCU it's mostly pinouts and I2C addresses.

Reply:
Sure.